docs/user/openstack/install_upi.md

```diff
@@ -367,6 +367,7 @@ data["networking"]["machineNetwork"] = machine_net
 data["platform"]["openstack"]["apiVIPs"] = api_vips
 data["platform"]["openstack"]["ingressVIPs"] = ingress_vips
 data["platform"]["openstack"]["controlPlanePort"] = ctrl_plane_port
+data["featureSet"] = "TechPreviewNoUpgrade"
```

Comment:
We should not activate TPNU for all UPI deployment, but instead document it as a requirement if the user wants to deploy with dual-stack.